packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -108,6 +108,7 @@ export const DataTableComponent = <T extends RecordWithId>({
         flexDirection: 'column',
         height: '100%',
         overflowX: 'unset',
+        tableLayout: 'fixed',
```

Comment:
a possible fix for the percentage column widths problem (#1140) which didn't actually help.

Reply:
In the CSS inspector, it informs me: `table-layout has no effect on this element since it’s not a table.`

It would need to be on the `<MuiTable>` component to have an effect, you've got it on the `TableContainer` component (which is just a `div`).

But personally, I'm not so in favour of fixed column widths anyway -- I think it's nicer to let content dictate widths and just set maximums.

Reply:
Doh! and thanks - have removed it.